torch/quantization/fake_quantize.py

```diff
@@ -64,6 +60,18 @@ def extra_repr(self):
             self.fake_quant_enabled, self.observer_enabled,
             self.scale, self.zero_point)
 
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(FakeQuantize, self)._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + 'scale'] = self.scale
```

Comment:
I'd add a comment that we need it because register_buffers doesn't support non-tensors yet (so it'd be easier to fix in the future)